File: Web Angular/src/app/models/Carrito.ts
import {Articulo} from "./Articulo";

export class Carrito {
  iD_Carrito: number = 0;
  iD_Cliente: number = 0;
  iD_Articulo: number = 0;
  cantidad_Articulo: number = 0;
  precio_Articulo: number = 0;
  Categoria: string = "";
  Modelo: string = "";
  Url_Imagen: string = "";
  articulo?: Articulo;
}

File: API ASP.NET Core/TiendaRopaOnline/Models/Carrito.cs
namespace TiendaRopaOnline.Models
{
    public class Carrito
    {
        public int iD_Carrito { get; set; }
        public int iD_Cliente { get; set; }
        public int iD_Articulo { get; set; }
        public int cantidad_Articulo { get; set; }
        public float precio_Articulo { get; set; }
    }
}
